src/Symfony/Component/Routing/Tests/RouteTest.php

```diff
@@ -136,7 +136,6 @@ public function getInvalidRequirements()
     {
         return [
            [''],
-           [[]],
```

Comment:
The test checks for an `InvalidArgumentException`, but we receive a `TypeError` now.